.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- 
+Thanks for contributing a pull request! Please ensure that your PR satisfies the checklist before submitting:
```

Comment:
Links can overflow the text box but I think nobody really cares to scroll left so we might want to break those long text lines.

Reply:
Fixed that in this document. The rest looked ok. Thanks!